### PR TITLE
Minor textual clarification to use the authority organization. 

### PR DESCRIPTION
**Pull request by rolandgroen**
The original text was ambiguous, it also could refer to the SP.

### CODE REVIEW

**Comment by rolandgroen** [comment]
Should be reviewed by  @hietkamp

File: kik-v/1-setup-network.md [hunk 1]
```diff
@@ -78,14 +78,14 @@ When you name an organisation, the service provider issues a NutsOrganizationCre
...
-The data producer needs to trust the validated query credential of the authority. This trust is registered explicitly. 
+The data producer needs to trust the validated query credential of the authority organization. This trust is registered explicitly. 
 
 ```http request
 POST http://localhost:3323/internal/vcr/v2/verifier/trust
 Content-Type: application/json
 
 {
-    "issuer": "did:nuts:<the did of the authority>",
+    "issuer": "did:nuts:<the did of thee Authority KIK-V organization>",
```
Comment (hietkamp) [inline kik-v/1-setup-network.md:88]: Bevat nog een typo

Comment (reinkrul) [inline kik-v/1-setup-network.md:88]:
```suggestion
    "issuer": "did:nuts:<the did of the Authority KIK-V organization>",
```

**Review by reinkrul — APPROVED** [review]

**Comment by reinkrul** [comment]
Thanks for the contribution @rolandgroen ! 🚀

**Pull request merged**